### DISCUSSION TITLE
Customize the message sended and add received confirmation - Feature request

### DISCUSSION THREAD

**Discussion by zzseba78** [craftcms/contact-form#219] [Q&A]
### Description

This is a question or a feature request if doesn´t exist yet: is there a way to customize the message structure and design with a template?

And it would be nice to be able to send a confirmation email to the sender with a customizable template too.

Thanks.

**Comment by brandonkelly** [comment]
You can customize the design if you are running Craft Pro, by specifying a custom email template in **Settings** → **Email**.

**Reply by brycekirk** [reply]
I have Craft Pro, and I've set an email template in Settings -> Email.  System emails are all successfully sent with my new template, but Craft Contact Form ignores it. Am I missing a setting somewhere??
